docs/ide-usage.md

```diff
@@ -88,59 +88,30 @@ The following plugins are needed:
 Supported IntelliJ versions: IntelliJ IDEA Ultimate Edition (required to work with Google App Engine).
 You can sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a student registered in an educational institution.
 
-**NOTE**
-> If you are migrating from Eclipse, you should delete `.project`, `.classpath`, `.launches`,
-> `src/main/webapp/WEB-INF/classes`, `src/main/webapp/WEB-INF/lib` and `src/main/webapp/WEB-INF/appengine-generated`.\
-> You can also copy `local_db.bin` from `src/main/webapp/WEB-INF/appengine-generated` to the location your dev server in
-> IntelliJ reads the datastore from after your IntelliJ setup. See [dev server setup](development.md#with-intellij) for
-> more details.
-
 ### Prerequisites
-1. You need a Java 7 SDK with the name `1.7` defined in IntelliJ IDEA as follows:
+1. You need a Java 8 SDK with the name `1.8` defined in IntelliJ IDEA as follows:
```

Comment:
I tried importing just now and this step is not necessary. This is "setting the default" which I don't really encourage (I mentioned "Note that the JDK to be used is not required to be the `default`" in the Eclipse setup).

Reply:
This isn't really setting the defaults (in the way you mean) but I think it is a good point you made, it looked like it is setting the defaults. I think IntelliJ interprets SDKs as `Project and Projects Defaults` as opposed to just `Project Defaults`, it might be a little confusing but I'm not sure if the instructions can explain this anywhere.

Reply:
And I'm not sure if you caught it the instructions were specifically labeled to require exact names because at least currently the `TestNG` run configurations use an alternative JDK rather than project JDK, which requires specifying the project JDK name.

That was due to the `TestNG` run configs in IntelliJ configured in a different way in the past, I'm actually trying to change that and hopefully we have no other settings that require linking to non-project SDKs.

Here's where I'm actually changing the run configurations, #7791. I'm actually upgrading the scope of the issue, so I wanted to resolve some other things in the PR as well.

Reply:
@LiHaoTan not sure I get what you mean. I can run any single unit test (can't run anything that needs dev server) or any of the launch configs just fine.

Reply:
@LiHaoTan I do have [this](https://drive.google.com/file/d/0B1ERLjVZ-gOVSUJ2NTAxY2kzams/view?usp=sharing) (GitHub's image upload feature is down for some reason) set up and I think that's what you need to do instead of "Project Defaults". Notice the parallel with the Eclipse's setup: 1.8 needs to be there, but not required to be the default.

Reply:
> If you have no current projects open, click Configure → Project Defaults → Project Structure.

This is what you mean by configuring project defaults right?

> Select SDKs in Platform Settings and check if there is a SDK named `1.8` with a JDK home path pointing to a JDK 8 path. Otherwise add a new SDK using JDK 8 with a name of `1.8`.

But if you look at the next step, I'm really not configuring any project defaults, but the **projects**(platform) defaults. 

The screenshot you showed me is exactly what the instructions are telling you to do (but with the confusing **defaults** name when you are not setting **defaults**)

Reply:
@LiHaoTan ok, it's my mistake. Turns out I've added the `1.8` config for a while ago for my own side project and coincidentally the name is the same.
@whipermr5 you can ignore this entire thread.

Reply:
@whipermr5 I just got the idea of uploading image helpers (much like Eclipse setup) though. You can use the above and the following two: [here](https://drive.google.com/file/d/0B1ERLjVZ-gOVd1dmbktlMy11aHc/view?usp=sharing) and [here](https://drive.google.com/file/d/0B1ERLjVZ-gOVbHNhN2tONlJQZkk/view?usp=sharing), and maybe more if you know where to put.

Reply:
@wkurniawan07 Sorry didn't see that you replied, I edited my comment a little to provide more context, I think I need to document this somewhere because I think there is a valid point being raised here, but I'm just not sure how.